app/views/govuk_publishing_components/components/_layout_footer.html.erb

```diff
@@ -102,6 +116,18 @@
         data-track-label="https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/"
         data-track-options='{"dimension29": "© Crown copyright"}'
         data-track-links-only
+        data-ga4-link="<%= {
+            "event_name": "navigation",
```

Comment:
Nitpick: indentation looks like 4 chars instead of 2?

Reply:
@andysellick Indentation is definitely a weakness of mine!